src/fields/config/types.ts

```diff
@@ -186,6 +186,7 @@ export type RichTextField = FieldBase & {
     placeholder?: string
     elements?: RichTextElement[];
     leaves?: RichTextLeaf[];
+    editableProps?: object;
```

Comment:
Can we leverage types from within Slate here? To give Payload devs a bit more intellisense?